src/clusterfuzz/_internal/tests/appengine/handlers/home_test.py

```diff
@@ -33,10 +33,12 @@ def setUp(self):
         'libs.access.get_access',
         'clusterfuzz._internal.base.external_users.allowed_jobs_for_user',
         'libs.helpers.get_user_email',
-        'clusterfuzz._internal.google_cloud_utils.storage.read_data'
+        'clusterfuzz._internal.google_cloud_utils.storage.read_data',
+        'clusterfuzz._internal.google_cloud_utils.storage.exists'
     ])
 
-    self.mock.read_data.return_value = ""
+    self.mock.read_data.return_value = ''
+    self.mock.exists.return_value = False
```

Comment:
Set this to `True` in `tests_results_external` so that for that case we have some tests of the code path were introspector reports does exist. `test_results_internal` can continue to test this with this returning False.
 
We'll also need to set a `read_data` with some fake data too in that test.

Reply:
Done.